src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen, act, waitFor, fireEvent } from '@testing-library/react';
+import { UserInfoDesktopView } from 'people/userInfo/UserInfoDesktopView';
+import nock from 'nock';
+import { user } from '__test__/__mockData__/user';
+import MockStoreEnvironment from '../../../__test__/__mockStore__/MockStoreEnvironment';
+import { MOCK_ENVIRONMENT_HOOKS } from '__test__/__mockStore__/constants';
+
+describe('Test for UserInfoDesktopView', () => {
+  beforeEach(() => {
+    nock(user.url).get('/person/id/1').reply(200, {});
+  });
+
+  const setShowSupport = jest.fn();
+
+  it('Test that user profile photo is visible', async () => {
+    render(
+      <MockStoreEnvironment>
+        <UserInfoDesktopView setShowSupport={setShowSupport} />
+      </MockStoreEnvironment>
+    );
+    const profileImg = screen.getByTestId('profile-img');
+    expect(profileImg).toBeInTheDocument();
+    expect(profileImg).toBeVisible();
+  });
+
+  it('Test that user connect QR button is rendered and upon clicking it renders connect modal on page', async () => {
+    const user = render(
+      <MockStoreEnvironment>
+        <UserInfoDesktopView setShowSupport={setShowSupport} />
+      </MockStoreEnvironment>
+    );
+
+    const qrIcon = screen.getByText('qr_code_2');
+    expect(qrIcon).toBeInTheDocument();
+    await act(async () => {
+      user.getByText('qr_code_2').click();
+      await waitFor(() => {
+        const modal = user.getByTestId('connect-modal');
+        expect(modal).toBeInTheDocument();
+        expect(modal).toBeVisible();
+      });
+    });
+  });
+
+  it('Test that description/bio for a user is visible if provided', async () => {
+    render(
+      <MockStoreEnvironment>
+        <UserInfoDesktopView setShowSupport={setShowSupport} />
+      </MockStoreEnvironment>
+    );
+    const description = screen.getByTestId('user-description');
+    expect(description).toBeInTheDocument();
+    expect(description).toBeVisible();
+  });
+
+  it('Test that connect are rendered and open connect modal upon clicking', async () => {
+    const user = render(
+      <MockStoreEnvironment>
+        <UserInfoDesktopView setShowSupport={setShowSupport} />
+      </MockStoreEnvironment>
+    );
+
+    expect(user.getByText('Connect')).toBeInTheDocument();
+
+    await act(async () => {
+      user.getByText('Connect').click();
+      await waitFor(() => {
+        const modal = user.getByTestId('connect-modal');
```

Comment:
lets assert that the text on the modal appears

`Discuss this bounty with` `Connect with Sphinx` ect
![Screenshot 2024-02-29 at 3 52 25 PM](https://github.com/stakwork/sphinx-tribes-frontend/assets/15950706/cbc54d3c-77dd-4deb-b3e8-7f55db1a1a26)